### PR TITLE
GNOME Shell 3.38 fixes and more...

### PR DESCRIPTION
**Pull request by Tamas-Toth-ebola**
Hi @lachhebo !

Finally I had some time to fix the extension to be compatible with latest GNOME Shell as StBoxLayout's child meta was deprecated in 3.36 and removed in 3.38. _(commented in the source code)_. I made the required fixes and also made some changes to make a little better visibility on my earlier planned extensions.gnome.org icon.

For clearer background I made a discrete release for the changes on GH: https://github.com/Tamas-Toth-ebola/corona-tracker/releases _(you can check the 'detailed' changes there (extension version info in the meta file is also bumped to 10 to proper upgrade))_

**Please if it possible and you like my changes...**
- accept my **PR**
- make a same like GH **tag** + **release** as mine _(as in the extensions.gnome.org description I referred to your corresponding release page)_
- change the extension's **description on extensions.gnome.org** to the string available in the EXTENSIONS.GNOME.ORG.md _(in description part)_
- change the extension's **icon on extensions.gnome.org** for the /resources/extensions.gnome.org.png _(what I also mention in the EXTENSIONS.GNOME.ORG.md)_

Hope you like my additions. Have a nice further day and keep yourself in safe!

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -1,8 +1,15 @@
 {
-  "name": "corona-tracker",
-  "uuid": "corona-tracker@lachhebo.github.io",
+  "_generated": "Generated by SweetTooth, do not edit",
```
Comment (lachhebo) [inline metadata.json]: i think we don' need the _generated parameter

**Comment by lachhebo** [comment]
Hi @Tamas-Toth-ebola, 

Thanks a lot for your work.
I will test it, merge it and deploy the new version to gnome extensions website.

**Review by lachhebo — APPROVED** [review]

**Comment by lachhebo** [comment]
By the way, don't foget to resolve the conflict, i need it to to merge the pr

File: metadata.json (hunk 1 above)
Comment (Tamas-Toth-ebola) [reply metadata.json]: Yes! This was not mine so I left it. But now as you told that we don't need I deleted in the conflict resolve.

**Comment by Tamas-Toth-ebola** [comment]
I hope I'm already done with everything. Just tell me if you need any other.

**Pull request merged**